digitalpalireader/content/js/send.js

```diff
@@ -331,29 +355,6 @@ function sendAnalysisToOutput(input, divclicked, frombox, add){
 	if(window.getSelection().toString())
```

Comment:
nice catch.

while you are at it, can you please also get these two lines into the flow for isWeb. I am pretty sure something else we dont know is missing yet, will light up.

regarding the isWeb, there is an approved item for beta that is for removing these conditions across the codebase as we dont need to support xul anymore.

Reply:
I was considering them but since I'm new to the project I wasn't sure they're needed in the Web version. Can you confirm they're needed? 

What I found so far is that the `add==='right'` check is only needed in the XUL version as there is a right click context menu (thus 'right'). Same goes about the `window.getSelection().toString()`: it seems it's related to the extension functionality and probably isn't needed in the Web version.

Maybe @yuttadhammo should join this mini-conversion to shed light.

Here's a screenshot of the Palemoon extension's right click functionality for reference:
![image](https://user-images.githubusercontent.com/61728359/76159834-fed44180-612c-11ea-8374-17057f6e7409.png)

Reply:
Confirming that they are needed. Lets retain the parity with XUL code. It should have been done initially.